pkg/resolver/search.go

```diff
@@ -467,9 +566,11 @@ func WhereClauseFilter(input *model.SearchInput) []exp.Expression {
 }
 
 func getKeys(stringArrayMap map[string][]string) []string {
-	keys := make([]string, 0, len(stringArrayMap))
+	i := 0
+	keys := make([]string, len(stringArrayMap))
 	for k := range stringArrayMap {
-		keys = append(keys, k)
+		keys[i] = k
+		i++
```

Comment:
```golang
keys := make([]string, len(stringArrayMap))
for k, i := range stringArrayMap {
	keys[i] = k
}

Reply:
But `i` is not an int in this case, right?

Reply:
🤦 sorry, I didn't realize it's a map.